app/test_main.py

```diff
@@ -1,3 +1,61 @@
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (-10, -100, [0, 0]),
+        (1_000_000, 1_000_000, [249996, 199997])
+    ],
+    ids=[
+        "for 0 years cat and dog 0 cat and dog are 0 years old",
+        "cat and dog are 15 years old",
+        "cat and dog are 23 years old",
```

Comment:
The test case on this line is duplicated with the one on line 22. Please remove one of them to avoid repetition in the tests. (Don't Repeat Yourself principle)